services/QuillLMS/client/app/bundles/Shared/components/shared/tooltip.tsx

```diff
@@ -1,6 +1,10 @@
 import * as React from 'react'
+import marked from 'marked';
 
 interface TooltipProps {
+  isMarkdown?: boolean,
```

Comment:
I think instead of handling the conversion from Markdown in this file, it would be better to do the conversion in the parent and pass it in as a prop, since we don't really have reason to think we're going to want to be doing this all that frequently.